plugins/source/hubspot/resources/services/crm/companies.go

```diff
@@ -0,0 +1,48 @@
+package crm
+
+import (
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func Companies() *schema.Table {
+	return &schema.Table{
+		Name:        "hubspot_crm_companies",
+		Description: "https://developers.hubspot.com/docs/api/crm/companies",
+		Resolver:    fetchCompanies,
```

Comment:
Can we change the tables to use:
https://github.com/cloudquery/plugin-sdk/blob/281e706260eb83e70992e672064e4f35eb51ab4b/transformers/struct.go#L116

It would mean less code and also if an SDK update adds a new field we won't need to manually add it.

Also it has a `WithPrimaryKeys` option so you don't need to redefine the `id` column